article-store/articles/tests/test_api/test_article_api.py

```diff
@@ -227,7 +235,7 @@ def test_can_get_article_content_without_giving_part_name(self,
                                                               content_es_front: Content,
                                                               article_0065_es_front_xml: str):
 
-        url = f'/articles/{article.id}/latest'
+        url = f'/articles/{article.id}/versions/latest'
```

Comment:
changes like these should usually be a separate PR?

Reply:
not that the history of the walking skeleton itself should be that clean in the long run, projects are going to be extracted in other repositories

Reply:
Yes I agree.